compiler/ztests/sql/union-distinct.yaml

```diff
@@ -0,0 +1,11 @@
+spq: |
+  select * from (values (1),(2),(3))
+  union distinct
+  select * from (values (2),(3),(4))
+  | sort c0
+
```

Comment:
Can we enable vectors for this test?

Reply:
Good catch. Done in 92d8ba3bdd05d34ecf1d1cfec8e579439f57da19.